.github/CODEOWNERS

```diff
@@ -1,2 +1,4 @@
 # Default owner for all directories not owned by others
-* @googleapis/yoshi-go-admins @yoshi-approver
+* @googleapis/yoshi-go-admins
+.github/auto-approve.yml @googleapis/github-automation
```

Comment:
codeowner needs explicit write access to the repo.

We no longer require this with the v2 schema that you are using

Reply:
Oh I just granted perms. Ill revoke and revert this